serverless/test/forwarder.spec.ts

```diff
@@ -353,10 +358,75 @@ describe("shouldSubscribeLogGroup", () => {
       [logGroupName]: { logGroup: { logGroupName } },
     });
     const shouldSub = await shouldSubscribeLogGroup(cloudWatchLogs as any, logGroupName);
-    expect(shouldSub).toBeTruthy();
+    expect(shouldSub).toBe(true);
   });
-
-  it("returns false if there is an existing subscription", async () => {
+  it("returns false if the log group only has 1 existing datadog-serverless-macro-filter subscription filter.", async () => {
```

Comment:
It looks like most of the other tests in this file don't end their descriptions with a `.` -- I would remove yours to keep it consistent